src/site/content/en/blog/reporting-observer/index.md

```diff
@@ -0,0 +1,216 @@
+---
+title: Know your code health
+subhead: Find deprecated APIs in your production apps
+authors:
+  - ericbidelman
+date: 2019-08-21
+updated: 2020-05-27
+hero: hero.jpg
+alt: A reporter observing something.
+description: |
+  `ReportingObserver` lets you know when your site uses a deprecated API or runs
+  into a browser intervention. The basic functionality originally landed in Chrome
+  69. As of Chrome 84, it can be used in workers. It's pretty simple.
+tags:
+  - blog # blog is a required tag for the article to show up in the blog.
+---
+
+`ReportingObserver` lets you know when your site uses a deprecated API or runs
+into a [browser intervention][interventions]. The basic functionality originally
+landed in Chrome 69. As of Chrome 84, it can be used in workers. It's pretty
+simple to use as you can see from the code sample below:
+
+```js
+const observer = new ReportingObserver((reports, observer) => {
+  for (const report of reports) {
+    console.log(report.type, report.url, report.body);
+  }
+}, {buffered: true});
+
+observer.observe();
+```
+
+Use the callback to send reports to a backend or analytics provider for
+analysis.
+
+Why is that useful? Until this API, deprecation and intervention warnings were
+only available in the DevTools as console messages. Interventions, in
```

Comment:
s/the DevTools/DevTools/ (right?)

Reply:
```suggestion
only available in DevTools as console messages. Interventions, in
```
@philipwalton y u no use the [Suggestion UI](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request) (step 6)

Reply:
Fixed.